package.json

```diff
@@ -11,6 +11,7 @@
     "eslint": "eslint --max-warnings 0 --cache --ext .jsx,js,tsx,ts packages",
     "build": "./build.js",
     "migrate": "node migrate",
+    "upload-ckeditor-bundle": "node deployCkEditorBundle",
```

Comment:
Does this refer to a file that missed getting `git add`ed?

Reply:
Whoops, it sure does.